crypto/fipsmodule/ec/ec.c

```diff
@@ -1095,6 +1180,15 @@ void EC_GROUP_set_point_conversion_form(EC_GROUP *group,
       form != POINT_CONVERSION_COMPRESSED) {
     abort();
   }
+  // Only dynamically allocated groups are mutable.
```

Comment:
how does this comment relate to the conversion form you're setting two lines below?